app/helpers/pageflow/social_share_helper.rb

```diff
@@ -24,8 +24,17 @@ def social_share_page_title(page)
     end
 
     def social_share_page_description(page)
-      return page.configuration['text'] if page.configuration['text'].present?
-      return page.configuration['description'] if page.configuration['description'].present?
+      return strip(page.configuration['text']) if page.configuration['text'].present?
+      return strip(page.configuration['description']) if page.configuration['description'].present?
+      social_share_entry_description(page.chapter.revision)
```

Comment:
I'd prefer adding an `entry` parameter to the method. Looking up the revision like this might trigger separate database accesses. Moreover, this only works since the passed revision has the same duck-type as the `entry` expected by `social_share_entry_description` method. Still `Revision#summary` does not need to have the name behavior as `PublishedEntry#summary`.